src/resources/assets/js/modules/upload.js

```diff
@@ -172,6 +172,12 @@ export default {
                 sending: function (file, xhr, formData) {
                     uploadProgress += parseFloat(100 / allFiles)
                     manager.progressCounter = `${Math.round(uploadProgress)}%`
+                    
+                    //fix if Dropzone doesnt find the form. This only happens if you use direct drag and drop on the div (__stack-container).
+                    if (!formData.has('upload_path')) {
+                        let path = document.getElementById('new-upload').getRootNode().getElementsByName('upload_path')[0].value;
+                        formData.append('upload_path', path);
```

Comment:
i think we should also check if the `path` is not empty b4 appending it.

Reply:
Added the check :)